From 7b51c421ddedfa8149a51fa755792c2daeb82c85 Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Sun, 24 Jan 2021 20:51:37 -0800
Subject: [PATCH 061/150] soc: ti: pruss: Correct the pruss_clk_init error
 trace text

[ Upstream commit f97a8a34353e993f7dc2c204faaa033955907bdc ]

The pruss_clk_init() function can register more than one clock.
Correct the existing misleading error trace upon a failure within
this function.

Signed-off-by: Suman Anna <s-anna@ti.com>
Signed-off-by: Santosh Shilimkar <santosh.shilimkar@oracle.com>
[s-anna@ti.com: cherry-pick commit 'f97a8a34353e' from v5.12]
---
 drivers/soc/ti/pruss.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/ti/pruss.c b/drivers/soc/ti/pruss.c
index aa3aba80dbc6..dc94335fc351 100644
--- a/drivers/soc/ti/pruss.c
+++ b/drivers/soc/ti/pruss.c
@@ -273,7 +273,7 @@ static int pruss_probe(struct platform_device *pdev)
 
 	ret = pruss_clk_init(pruss, child);
 	if (ret) {
-		dev_err(dev, "failed to setup coreclk-mux\n");
+		dev_err(dev, "pruss_clk_init failed, ret = %d\n", ret);
 		goto node_put;
 	}
 
-- 
2.34.1

